src/Abp.HangFire/Hangfire/AbpHangfireAsyncAuthorizationFilter.cs

```diff
@@ -30,7 +30,7 @@ public AbpHangfireAsyncAuthorizationFilter(string requiredPermissionName = null)
         /// <returns></returns>
         public async Task<bool> AuthorizeAsync(DashboardContext context)
         {
-            return !_requiredPermissionName.IsNullOrEmpty() && IsLoggedIn() && await IsPermissionGrantedAsync(_requiredPermissionName);
+            return IsLoggedIn() && (_requiredPermissionName.IsNullOrEmpty() && await IsPermissionGrantedAsync(_requiredPermissionName));
```

Comment:
@oguzhanagir Why was this changed? As all conditions need to be met (&&) and we do not need to resolve the AbpSession service if there is no requiredPermissionName, which is more expensive than the `IsNullOrEmpty()` condition.

Reply:
I think we should use the same controls in AbpHangfireAuthorizationFilter. It covers when the case when we want to show dashboard to all loggedin users. @antheus-s what do you think ?

Reply:
@ismcagdas I see what you mean, but that would mean that the code between parentheses should be an `||` instead of `&&`.

Reply:
Yes, that is correct. I will make this change and commit to dev branch.